index.html

```diff
@@ -7,8 +7,9 @@
   <title>Awesome Books</title>
   <link rel="stylesheet" href="styles.css">
   <link rel="stylesheet" href="https://bootswatch.com/5/flatly/bootstrap.min.css">
-  <script src="app.js" defer></script>
-  <script src="https://cdn.jsdelivr.net/npm/luxon@1.27.0/build/global/luxon.min.js"></script>
+  <link rel="shortcut icon" href="./assets/book-icon.png" type="image/x-icon">
+  <script src="app.js" type="module" defer></script>
```

Comment:
- [ ]  Great Job :clap:  but The entry point for your JavaScript code should be called index.js and not app.js, kindly rename your entry point name into `index.js`